translations/fr/get5.phrases.txt

```diff
@@ -252,6 +344,26 @@
     {
         "fr"            "Vous avez bougé à la position de coach, car votre équipe est complète."
     }
+    "CannotLeaveCoachingTeamIsFull"
+    {
+        "fr"            "Vous ne pouvez pas quitter votre position de coach car votre équipe est complète."
+    }
+    "CoachingNotEnabled"
+    {
+        "fr"            "Le coaching n'est pas activé. Vous devez définir {1} à 1."
+    }
+    "PlayerIsCoachingTeam"
+    {
+        "fr"            "{1} coach {2}."
```

Comment:
Is this right? Just `PlayerName coach Team A`?

Reply:
Yes